geode-core/src/main/java/org/apache/geode/cache/wan/GatewaySender.java

```diff
@@ -174,7 +174,15 @@
    */
   int GET_TRANSACTION_EVENTS_FROM_QUEUE_RETRIES =
       Integer.getInteger(GeodeGlossary.GEMFIRE_PREFIX + "get-transaction-events-from-queue-retries",
-          10);
+          3);
+  /**
+   * Milliseconds to wait before retrying to get events for a transaction from the
+   * gateway sender queue when group-transaction-events is true.
+   */
+  int GET_TRANSACTION_EVENTS_FROM_QUEUE_WAIT_TIME_MS =
```

Comment:
Please use SystemPropertyHelper for setting system properties....
E.g.:
SystemPropertyHelper
        .getProductIntegerProperty(SystemPropertyHelper.EVICTION_SCAN_THRESHOLD_PERCENT);

Look for use of VICTION_SCAN_THRESHOLD_PERCENT.